lib/types/api.d.ts

```diff
@@ -25,7 +25,10 @@ export interface IMap<T> {
     [key: string]: T | null;
 }
 
-export interface IRazorpayQuery {
+type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>
```

Comment:
Is this Omit doing anything different? If not TypeScript has built-in support for Omit<> you don't need to redefine it.